dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtility.java

```diff
@@ -177,7 +177,7 @@ static DefaultRepositorySystemSession newSessionForVerboseListDependency(
         new AndDependencySelector(
             // ScopeDependencySelector takes exclusions. 'Provided' scope is not here to avoid
             // false positive in LinkageChecker.
-            new ScopeDependencySelector("test"),
+            new ScopeDependencySelector("test", "system"),
```

Comment:
system scope is a different thing. I'm not sure we want to exclude everything with system scope.

Reply:
Good. I'll exclude jdk groupId for now.